### ISSUE TITLE
Unable to pass the file_path

### ISSUE THREAD

**Issue by kasapu** [microsoft/Form-Recognizer-Toolkit#29]
I can't pass documents from my local desktop and unable to provide Url or file_path

**Comment by luzhang06** [comment]
@kasapu can you please specify which sample code you are running into this issue?

**Comment by kasapu** [comment]
Hi @luzhang06 thanks for your response, I am using code in the attached path "SampleCode/Python/sample_rag_langchain.ipynb" & https://github.com/microsoft/Form-Recognizer-Toolkit/blob/0fbbc55826df93018cc9b2bc9e517b1c1c8d90de/SampleCode/Python/sample_rag_langchain.ipynb Thank You @luzhang06

**Comment by luzhang06** [comment]
@kasapu can you please share what error you see with this sample code? We support both local file and public URL as input. You can learn more here: https://python.langchain.com/docs/integrations/document_loaders/azure_document_intelligence.

**Comment by kasapu** [comment]
@luzhang06, thank you for your response. I am trying to upload file but I am getting error as shown in below image. I tried data path and file path options. I checked file in folder and it is project folder and file is not opened. Please help me. Thank You Zhang.
![image](https://github.com/microsoft/Form-Recognizer-Toolkit/assets/26332107/34af492e-d5c9-439b-9c8c-1cb842b9d0f2)

**Comment by luzhang06** [comment]
@toothache any comment on this?

**Comment by luzhang06** [comment]
@kasapu it seems you are passing in the relative path which the file cannot be found. Have you tried the absolute path?

**Comment by kasapu** [comment]
Indeed, @luzhang06, I have passed the absolute path along but it hasn't worked as expected. I will make sure to send an email to the formrecog for further analysis. In the meantime, could you please provide me with the AI doc intelligence with any other framework GitHub repository link? I need it to demonstrate Azure AI Document Intelligence capabilities integrated with RAG to our clients. Thank You for your support and messages.

**Comment by luzhang06** [comment]
This is the most relevant one you can find. Other repos are not leveraging markdown output yet but you can probably have a try: 

SDK and REST API
[Quickstart: Document Intelligence SDKs](https://learn.microsoft.com/en-us/azure/ai-services/document-intelligence/quickstarts/get-started-sdks-rest-api?view=doc-intel-4.0.0&pivots=programming-language-python#layout-model) – use your preferred SDK or REST API to extract content and structure from documents.
[Sample code](https://aka.ms/layout-md-sample) of using Layout API to output in markdown format.
 

Build “chat with your document” with semantic chunking
This [cookbook](https://aka.ms/doc-gen-ai) shows a simple demo for RAG pattern with Azure AI Document Intelligence as document loader and Azure Search as retriever in Langchain.
This [solution accelerator](https://github.com/Azure-Samples/chat-with-your-data-solution-accelerator) demonstrates an end-to-end baseline RAG pattern sample that uses Azure AI Search as a retriever and Azure AI Document Intelligence for document loading and semantic chunking.

**Comment by kasapu** [comment]
Thank You much @luzhang06, I will try the links provided and will let you know if any. Thank You
